metric.go

```diff
@@ -173,14 +215,14 @@ type Metric interface {
 // NewMetric returns a metric with one fixed value that cannot be changed.
 //
 // NewMetric panics if the length of labelValues is not consistent with desc.labels().
-func NewMetric(desc MetricDesc, value float64, labelValues ...string) Metric {
+func NewMetric(desc MetricDesc, value float64, labelValues []string, userLabels ...*dto.LabelPair) Metric {
```

Comment:
Nit: if you;re making `labelValues` an array, you may as well do it for `userLabels`.

Reply:
i've done it this way, to make the last arg optional as you've had done it before